chapter_5_practic/task10.cpp

```diff
@@ -0,0 +1,40 @@
+// Write a function that, when you call it, displays a message telling how many times it has
+// been called : �I have been called 3 times�, for instance.Write a main() program that calls
+// this function at least 10 times.Try implementing this function in two different ways.
+// First, use a global variable to store the count.Second, use a local static variable.Which
+// is more appropriate ? Why can�t you use a local variable ?
+
+
+#include <iostream>
+
+void println_global();
+void println_local();
+
+int counter = 1;
```

Comment:
The variable keeps an irrelevant counter.
If the function was not called, it keeps 1 as a counter.